cmd/rookflex/cmd/mount.go

```diff
@@ -154,6 +154,12 @@ func mountDevice(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, devic
 			"Ignore error about Mount failed: exit status 32. Kubernetes does this to check whether the volume has been formatted. It will format and retry again. https://github.com/kubernetes/kubernetes/blob/release-1.7/pkg/util/mount/mount_linux.go#L360",
 			false)
 		log(client, fmt.Sprintf("formatting volume %v devicePath %v deviceMountPath %v fs %v with options %+v", opts.VolumeName, devicePath, globalVolumeMountPath, opts.FsType, options), false)
+		err = os.Chown(opts.MountDir, int(opts.PodUser), int(opts.PodGroup))
```

Comment:
After an upgrade the pod user and group would be 0 since they weren't set on the original volume attachment. Should we skip calling chown if they are both 0? Or will it just chown to root, which was already assumed before anyway?

Reply:
good question, it would just chown to root, which was already the default. I'm not sure if that means we should skip it or not